public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Probe Dictionary",
+  "name": "Firefox Probe Dictionary - Find Probes",
+  "icons": [
+    {
+      "src": "favicon.png",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
```

Comment:
Was this originally an *ico* file? The *type* looks wrong and I don't think the *sizes* property would apply to PNGs, either.